actionpack/lib/action_dispatch/middleware/exception_wrapper.rb

```diff
@@ -261,13 +261,13 @@ def build_backtrace
         end
 
         (@exception.backtrace_locations || []).map do |loc|
-          if built_methods.key?(loc.label.to_s)
+          if built_methods.key?(loc.base_label.to_s)
```

Comment:
```suggestion
          if built_methods.key?(loc.base_label)
```

Pretty sure `base_label` is always a String.

Reply:
It will be nil when the error is a SyntaxErrorProxy, but I think that's ok too.